web/packages/teleport/src/UnifiedResources/FilterPanel.tsx

```diff
@@ -167,8 +176,9 @@ const FilterTypesMenu = ({
         size="small"
         onClick={handleOpen}
       >
-        Type
+        Types {kindsFromParams.length > 0 ? `(${kindsFromParams.length})` : ''}
```

Comment:
Similarly, this will change once we add more filters but abstracting now seemed a bit frivolous